tracer/test/snapshots/DataStreamsMonitoringKafkaTests.HandlesFanIn.verified.txt

```diff
@@ -145,6 +145,13 @@
             topic:data-streams-fan-in-out-1,
             type:kafka_produce
           ]
+        },
+        {
+          Tags: [
+            partition:2,
+            topic:data-streams-fan-in-out-2,
+            type:kafka_produce
```

Comment:
Interesting 🤔 I guess this is because you started sending more messages in the fan in scenario?

Reply:
yes it'd the one extra message sent.